scripts/mbedtls_framework/build_tree.py

```diff
@@ -7,17 +7,24 @@
 
 import os
 import inspect
+import re
 from typing import Optional
 
 def looks_like_tf_psa_crypto_root(path: str) -> bool:
     """Whether the given directory looks like the root of the PSA Crypto source tree."""
-    return all(os.path.isdir(os.path.join(path, subdir))
-               for subdir in ['include', 'core', 'drivers', 'programs', 'tests'])
+    try:
+        with open(os.path.join(path, 'scripts', 'project_name.txt'), 'r') as f:
```

Comment:
I may have missed that discusion since I was away but is using a `project_name.txt` file the cannocnical and universal accepted way to convey infromation between the mbedtls branches?

Or is that a temporary workaround?

Reply:
`project_name.txt` has been introduced in #8609 to detect between Mbed TLS and TF-PSA-Crypto projects. I am not extending it here to be able to detect between branches of the projects because it would be yet another file to update when we change version.